frontend/src/metabase/visualizations/components/ChartSettings/BaseChartSettings/hooks.ts

```diff
@@ -25,21 +28,26 @@ export const useChartSettingsSections = ({
         sectionObj[widget.section].push(widget);
       }
     }
+    if (shouldShowChartTypeSelector) {
+      sectionObj[t`Chart`] = [];
```

Comment:
:x: new changes do not work in languages other than English, probably because of things like `chartSettingCurrentSection?.toLowerCase() === "chart";`

```suggestion
      sectionObj.chart = [];
```

Reply:
should be fixed at 14e33332065445807da3921bec97600855b1b8c9